plugins/device/api/req.go

```diff
@@ -15,3 +15,8 @@ const (
 type AddDeviceReq struct {
 	Devices []string `json:"devices"`
 }
+
+// PeerEditZoneReq structure
+type PeerEditZoneReq struct {
+	Zone string `json:"zone"`
```

Comment:
Zone can be part of `PeerEditReq` because Zone isn't set for devices but only peers. It doesn't have to be in the devices plugin.